pycbc/events/stat.py

```diff
@@ -240,7 +240,7 @@ def rank_stat_coinc(self, sngls_list, slide, step, to_shift,
         numpy.ndarray
             Array of coincident ranking statistic values
         """
-        cstat = sum(sngl[1] ** 2. for sngl in sngls_list) ** 0.5
```

Comment:
I think this change is going to break this statistic in the standard workflow, where this statistic did work before and did not have a ['snglstat'] element.

Here, I've not changed the previous behaviour (although one might argue a change might be needed), where you call through to the `single` method of the Stat class to get whatever that returns (which can be computed without knowing anything about the other ifo(s), and then feeding that through to `rank_coinc`. For the QuadratureSum method you have not changed the `single` method, but have changed  the `rank_coinc` method, this breaks this statistic.

If possible, PyCBC Live should be doing the same and not manually constructing the single_info as you do on line 1046 of coinc.py but instead using the `single` method of `self.stat_calculator` and then passing that along .... Maybe there's some reason why it needs to do this though??